packages/fxa-auth-server/lib/routes/utils/signup.js

```diff
@@ -65,6 +65,7 @@ module.exports = (log, db, mailer, push, verificationReminders, glean) => {
 
       // Our post-verification email is very specific to sync,
       // so only send it if we're sure this is for sync or sync scoped client.
+      // Do not send for desktop Relay even if the client is Firefox
```

Comment:
Just curious, why do you say 'desktop' here?

Reply:
In the future, we might support this flow in mobile too. Maybe we clarify with, "for browser sign-ins that are not Sync" or something like that in these comments?

Reply:
That's clearer, thanks @LZoog!